adsingestp/parsers/jats.py

```diff
@@ -938,21 +938,27 @@ def _parse_pubdate(self):
         for d in pub_dates:
             pub_format = d.get("publication-format", "")
             pub_type = d.get("pub-type", "")
+            date_type = d.get("date-type", "")
+            accepted_date_types = ["pub", "", "first_release"]
```

Comment:
This seems unsustainable to maintain a list like this as we add new journals, but I could not think of a better way for now. I'm open to suggestions.